### PR TITLE
Authentication setup done

### PR DESCRIPTION
**Pull request by developerKumar**

### CODE REVIEW

File: src/auth/facebook.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+// Setting up our Facebook strategy for passport
+passport.use(
+  new FacebookStrategy(
+    {
+      clientID: process.env.FB_ID, // found on your facebook app profile
+      clientSecret: process.env.FB_SECRET, // found on your facebook app profile
+      callbackURL:
+        process.env.FB_CALLBACK_URI ||
+        "http://localhost:8000/login-facebook/callback", // should be the base URI for your server followed `/login-facebook/callback`
```
Comment (anistark) [inline src/auth/facebook.js]: We can replace this with simply `/login-facebook/callback` if in-case someone wants to use a different PORT for their local system.

File: src/auth/facebook.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+      {
+        user: req.user
+      },
+      SECRET
+    );
+
+    // Pass JWT to the client side in a URL Query Param
+    res.redirect(
+      `${process.env.CLIENT_URI ||
+        "http://localhost:3000"}/?token=${signedToken}`
```
Comment (anistark) [inline src/auth/facebook.js]: Same as above. We should try to avoid specifying port and url at any point directly in code..

**Review by anistark — CHANGES_REQUESTED** [review]

File: src/auth/facebook.js (hunk 1 above)
Comment (developerKumar) [reply src/auth/facebook.js]: thanks, I will do

File: src/auth/facebook.js (hunk 2 above)
Comment (developerKumar) [reply src/auth/facebook.js]: thanks, I will do.

**Comment by developerKumar** [comment]
Merged to develop!

**Pull request closed without merging**